.travis.yml

```diff
@@ -1 +1,7 @@
 language: go
+
+go:
+  - "1.9"
```

Comment:
Are Go Modules backward-compatible? These CI versions might need some pruning.

Reply:
Not exactly no. But vendoring is the magic that makes this work. Please see https://github.com/golang/go/wiki/Modules#how-do-i-use-vendoring-with-modules-is-vendoring-going-away for more details. Also note the matrix is building all these versions just fine so things seem to line up fine.